e2etest/declarativeHelpers.go

```diff
@@ -174,26 +185,27 @@ var eOperation = Operation(0)
 type Operation uint8
 
 func (Operation) Copy() Operation        { return Operation(1) }
-func (Operation) Sync() Operation        { return Operation(2) }
-func (Operation) CopyAndSync() Operation { return eOperation.Copy() & eOperation.Sync() }
-func (Operation) Remove() Operation      { return Operation(3) }
+func (Operation) Sync() Operation        { return Operation(1<<1) }
+func (Operation) CopyAndSync() Operation { return eOperation.Copy() | eOperation.Sync() }
+func (Operation) Remove() Operation      { return Operation(1<<2) }
+func (Operation) Resume() Operation 	 { return Operation(1<<7) } // Resume should only ever be combined with Copy or Sync, and is a mid-job cancel/resume.
```

Comment:
just curious: Why 7?

Reply:
Last bit on a uint8, gives us plenty of room for future operations we may want to implement or test (e.g. CLFSLoad)